index.html

```diff
@@ -7,53 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="Oppia : A tool for creating interactive tutors">
-
     <link rel="stylesheet" type="text/css" media="screen" href="static/css/stylesheet.css">
-
+    <link rel="shortcut icon" type="image/png" href="static/images/favicon.png">
     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.25/angular.min.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.25/angular-route.js"></script>
     <script src="app.js"></script>
-
     <title ng-bind="$root.title"></title>
   </head>
 
   <body>
-    <div id="header_wrap" class="outer">
-      <header class="inner" style="max-width: 900px;">
-        <a id="forkme_banner" href="https://github.com/oppia/oppia">
-          View on GitHub
-        </a>
-        <h1 id="project_title">
-          <a href="https://www.oppia.org" class="anchor">Oppia</a>
-        </h1>
-        <h2 id="project_tagline" ng-bind="$root.projectTag"></h2>
-      </header>
-    </div>
-
-    <div id="main_content_wrap" class="outer">
-      <div>
-      <!--A container to make a grid with a fixed column-->
-        <div class="container row" style="width: 900px;">
-          <div ng-include="'pages/sidebar.html'"></div>
+    <div id="page_overlay"></div>
+    <div class="outer">
```

Comment:
This is actually fine. <div id="page_overlay"> and <div class="outer"> (line 19 and 20 in green) are replacing <div id="header_wrap" class="outer> from above (line 21 red). The deleted block above has the closing divs deleted below at (line 39 to line 41 red).